format/format.go

```diff
@@ -154,6 +155,15 @@ func sortEverything(node *yaml.Node) {
 	}
 }
 
+var tektonVariables = regexp.MustCompile(`\$\( (results|params|workspaces|credentials|context|steps)\b`)
+
+// FixTektonVariables ensures references to a Tekton variable do not contain a leading space as that
+// prevents Tekton from processing them. For example, "$( results" is replaced with "$(results"
+// https://tekton.dev/docs/pipelines/variables/#variables-available-in-a-task
+func FixTektonVariables(buf bytes.Buffer) string {
```

Comment:
This could have been private

Reply:
Planning on using it in the tash tool.

Reply:
Already included in the `Format` which is used by tash, so no need...